scripts/bring_assisted_service.sh

```diff
@@ -64,7 +65,11 @@ if [[ "${OPENSHIFT_CI}" == "true" && "${REPO_NAME}" == "assisted-service" && "${
   git rebase -v "origin/${PULL_BASE_REF}"
 else
   cd assisted-service
-  retry -- git fetch --force origin "${SERVICE_BASE_REF}:FETCH_BASE" "${SERVICE_BRANCH}"
-  git reset --hard FETCH_HEAD
-  git rebase FETCH_BASE
+  if [ "${SERVICE_COMMIT_REF}" != "" ]; then
```

Comment:
Can we exit early for each option? Maybe adding a comment of why each option is used?
Also I'd recommend not reusing just the `cd assisted-service` , but rather have each option on the same level for  clearity

Reply:
We can't exit early as it is called by other scripts as well

Reply:
Maybe I can wrap this in functions though

Reply:
Done